README.md

```diff
@@ -2,6 +2,7 @@ Intermediate Flask
 ==================
 
 By [Dan Schlosser](http://danrs.ch) and [ADI](https://adicu.com)
+Documentation Edits by [Cameron Yick](http://cameronyick.us)
```

Comment:
maybe "with help from" instead of "Documentation Edits"?  I don't think you give yourself enough credit!